src/guides/v2.3/graphql/mutations/set-payment-method.md

```diff
@@ -112,3 +112,14 @@ Attribute |  Data Type | Description
 {% include graphql/cart-object.md %}
 
 [Cart query output]({{page.baseurl}}/graphql/queries/cart.html#cart-output) provides more information about the `Cart` object.
+
+## Errors
+
+Error | Description
+--- | ---
+`Could not find a cart with ID "XXX"` | The specified `cart_id` value does not exist in the `quote_id_mask` table.
+`Required parameter "cart_id" is missing.` | The value specified in the `cart_id` attribute is empty.
+`Required parameter "code" for "payment_method" is missing.` | The value specified in the `code` attribute is empty.
```

Comment:
Hi @atwixfirster. We already had a chat on this matter, I just remind you. An "attribute" cannot have a value until it's a default value, i.e.

```
function example ($par = 99) 
```

If we are talking about passing a value to the function, we are talking about an "argument". So, to avoid confusion, it's better to be precise here. So

```suggestion
`Required parameter "code" for "payment_method" is missing.` | The value specified in the `code` argument is empty.
```
It applies to other mentions of attributes as well.

Thank you!

Reply:
**implemented**

Also I've updated "Errors" section for the next mutations:
- addConfigurableProductsToCart
- addDownloadableProductsToCart
- addSimpleProductsToCart
- addVirtualProductsToCart
- applyCouponToCart

Reply:
Hi @atwixfirster. Awesome, thank you. 

But could you create a separate PR with the corresponding fixes, please? Adjust the changes related to the current topic only within the scope of this PR.

Thank you!

Reply:
> But could you create a separate PR with the corresponding fixes, please?

**done** in https://github.com/magento/devdocs/pull/6664

Thank you!